Sources/Screens/DeveloperMenu/DeveloperMenuViewController.swift

```diff
@@ -101,6 +101,46 @@ final class DeveloperMenuViewController: BaseViewController {
             unhappyPathResultLabel.font = .bodyBold
         }
     }
+    
+    @IBOutlet private var unsuccessfulPathButton: RoundedButton! {
```

Comment:
Very minor point, but to make things easier for anyone new who joins, or for us to remember easily in future, would it be a good idea to add a comment to each of the unhappy paths to explicitly say which errors are called? Or rename the buttons/labels slightly?

Reply:
Yeah could probably be done, might rename to "Happy", "Error" and "Unauthorized" 👍